ui/src/shared/utils/resourceToTemplate.ts

```diff
@@ -0,0 +1,94 @@
+import _ from 'lodash'
+import {getDeep} from 'src/utils/wrappers'
+import {Task, Label} from 'src/types/v2'
+import {ITemplate, TemplateType} from '@influxdata/influx'
+
+const CURRENT_TEMPLATE_VERSION = '1'
```

Comment:
looks different then the `0.1.0` in the `blankTemplate` on line 8. Should they be the same?

Reply:
Thanks!